tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -711,6 +711,13 @@ def transform_param_to_str(value):
                                         'will be detected automatically for torch.nn.Module based model instances, '
                                         'for for scripted models may requires to set manually. Example of usage: for forward method defined as'
                                         ' def forward(self, x, y), it will be ["x", "y"]', '', '', None)
+    },
+    "paddle":
+    {
+    'example_input': ParamDescription('Sample of model input in original framework. '
+                                       'For PaddlePaddle it can be Paddle Variable.', '', '', None),
+    'example_output': ParamDescription('Sample of model output in original framework. '
```

Comment:
We removed this params dictionary, please add this description here in new PaddlePaddle group https://github.com/openvinotoolkit/openvino/blob/d70d8509c31de117b4f6bed2c435282ee9140355/tools/mo/openvino/tools/mo/convert.py#L84

There are two tests of params description. To make them pass change condition here  https://github.com/openvinotoolkit/openvino/blob/d70d8509c31de117b4f6bed2c435282ee9140355/tests/layer_tests/mo_python_api_tests/test_mo_help.py#L27
to
`if group == 'Pytorch-specific parameters:' or 'PaddlePaddle-specific parameters:':`

and add new param names here:
https://github.com/openvinotoolkit/openvino/blob/d70d8509c31de117b4f6bed2c435282ee9140355/tools/mo/unit_tests/mo/utils/cli_parser_test.py#L2036

Reply:
@popovaan 
Fix them as you mentioned.